local-cli/generator-wpf/templates/src/App.xaml.cs

```diff
@@ -36,6 +37,8 @@ protected override void OnStartup(StartupEventArgs e)
         /// <param name="arguments"></param>
         private void OnCreate(string[] arguments)
         {
+            LauncherModule.SetActivatedUrl(String.Join(" ", arguments));
```

Comment:
Originally `Linking.getInitialURL()` is for iOS/Android apps to support Deep Link, while in desktop application we don't have similar Deep Link URL concept.
I have no good idea but just to pass arguments as string. That's why passing arguments as initialURL getting a little weird here.

From my point of view is just to make sure `Linking.getInitialURL()` will throw exception and feel free to discuss if there are any better options to put in intialURL.